toko/apps/products/models.py

```diff
@@ -34,10 +36,14 @@ class Product(TimeStampedModel):
     is_active = models.BooleanField(default=True)
     created_by = models.ForeignKey('users.User', related_name='products',
                                    blank=True, null=True)
-    category = models.ForeignKey('products.Category', related_name='products',
-                                 blank=True, null=True)
     price = models.DecimalField(validators=[MinValueValidator(0)], blank=True,
                                 null=True, decimal_places=2, max_digits=1000)
+    categories = models.ManyToManyField('products.Category',
+                                        related_name='categories',
+                                        blank=True, null=True)
+    default = models.ForeignKey('products.Category',
```

Comment:
ini untunk nyoba aja lebih baik pake oneto manaay ataau many to many,
sepertinya lebih cocok many to many

On Thu, Feb 18, 2016 at 8:52 PM, Sutrisno Efendi <notifications@github.com>
wrote:

> In toko/apps/products/models.py
> <https://github.com/ular/toko/pull/7#discussion_r53315488>:
>
> >      price = models.DecimalField(validators=[MinValueValidator(0)], blank=True,
> >                                  null=True, decimal_places=2, max_digits=1000)
> > +    categories = models.ManyToManyField('products.Category',
> > +                                        related_name='categories',
> > +                                        blank=True, null=True)
> > +    default = models.ForeignKey('products.Category',
>
> Default ini untuk apa ya? Kayaknya gak perlu ada default category.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ular/toko/pull/7/files#r53315488>.
>